lib/Backend/Lower.cpp

```diff
@@ -4932,15 +4934,47 @@ Lowerer::LowerNewScObjArray(IR::Instr *newObjInstr)
         Assert(weakFuncRef);
     }
 
-    IR::Opnd *opndSrc1 = newObjInstr->UnlinkSrc1();
-    if (opndSrc1->IsImmediateOpnd())
+    IR::LabelInstr *labelDone = IR::LabelInstr::New(Js::OpCode::Label, func);
+    IR::Opnd *opndSrc = newObjInstr->GetSrc2();
```

Comment:
sure.

---
In reply to: [71648999](https://github.com/Microsoft/ChakraCore/pull/1304#discussion_r71648999) [](ancestors = 71648999)